core/src/validator.rs

```diff
@@ -1412,11 +1389,10 @@ impl Validator {
             let (sender, _receiver) = tokio::sync::mpsc::channel(1);
             (None, sender, None)
         } else {
+            runtimes.insert_runtime(RuntimeId::TurbineQuicEndpoint, 4);
```

Comment:
I think it would be nice to have all the runtime configuration parameters in the same location.
Though, if we want to keep the runtime construction as an optional operation, it would require additional wiring.
I still think it is something to consider.

Essentially, the idea is that `RuntimeRegistry::new()` might take all the parameters always.  Perhaps as a configuration struct, making it easy to see which runtimes will get how many threads.
And the `get_runtime_handle()` can then conditionally create a runtime it needs.